### ISSUE TITLE
[bug] critical vulnerabilities persistenceagent

### ISSUE THREAD

**Issue by deepk2u** [kubeflow/pipelines#8943]
### Environment

*  How did you deploy Kubeflow Pipelines (KFP)?
KFP is deployed via a 'full kubeflow deployment' on AWS
*  KFP version: 
1.6.1

### Steps to reproduce
A security scanning tool running against our kubeflow cluster detected that the persistenceagent pod is vulnerable
```
Vulnerabilities
===============
  CVE                 Package      Version       Severity    Status                 CVSS
  ---                 -------      -------       --------    ------                 ----
  CVE-2022-37434      zlib         1.2.11-r1     critical    fixed in 1.2.13-r0     9.8
  CVE-2018-1000517    busybox      1.28.4-r3     critical    fixed in 1.29.3-r10    9.8
  CVE-2021-30139      apk-tools    2.10.1-r0     high        fixed in 2.10.8-r0     7.5
  CVE-2019-5747       busybox      1.28.4-r3     high        fixed in 1.30.1-r5     7.5
  CVE-2018-25032      zlib         1.2.11-r1     high        fixed in 1.2.12-r3     7.5
  CVE-2018-20679      busybox      1.28.4-r3     high        fixed in 1.30.1-r5     7.5
  CVE-2021-42386      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42385      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42384      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42382      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42381      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42380      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42379      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42378      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42376      busybox      1.28.4-r3     medium      fixed in 1.34.1-r7     5.5
  CVE-2021-41581      libressl     2.7.5-r0      medium      fixed in 3.4.3-r0      5.5
  CVE-2020-28928      musl         1.1.19-r11    medium      fixed in 1.2.2-r4      5.5
  CVE-2021-42374      busybox      1.28.4-r3     medium      fixed in 1.34.1-r7     5.3

Compliance
==========
  Severity    Description
  --------    -----------
  high        (CIS_Docker_CE_v1.1.0 - 4.1) Image should be created with a non-root user

Vulnerability
    Critical: 2
    High:     12
    Medium:   4
    Low:      0
    Total:    18

  Compliance
    Critical: 0
    High:     1
    Medium:   0
    Low:      0
    Total:    1
```
### Expected result

Fix all CVEs and Compliance issues

### Materials and Reference

---

Impacted by this bug? Give it a 👍.

**Comment by deepk2u** [comment]
Found the same Vulnerabilities for scheduledworkflow pods as well

**Comment by deepk2u** [comment]
Found the same Vulnerabilities for cache-server pods as well

```
Vulnerabilities
===============
  CVE                 Package      Version       Severity    Status                 CVSS
  ---                 -------      -------       --------    ------                 ----
  CVE-2022-37434      zlib         1.2.11-r1     critical    fixed in 1.2.13-r0     9.8
  CVE-2018-1000517    busybox      1.28.4-r3     critical    fixed in 1.29.3-r10    9.8
  CVE-2021-30139      apk-tools    2.10.1-r0     high        fixed in 2.10.8-r0     7.5
  CVE-2019-5747       busybox      1.28.4-r3     high        fixed in 1.30.1-r5     7.5
  CVE-2018-25032      zlib         1.2.11-r1     high        fixed in 1.2.12-r3     7.5
  CVE-2018-20679      busybox      1.28.4-r3     high        fixed in 1.30.1-r5     7.5
  CVE-2021-42386      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42385      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42384      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42382      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42381      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42380      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42379      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42378      busybox      1.28.4-r3     high        fixed in 1.34.1-r7     7.2
  CVE-2021-42376      busybox      1.28.4-r3     medium      fixed in 1.34.1-r7     5.5
  CVE-2021-41581      libressl     2.7.5-r0      medium      fixed in 3.4.3-r0      5.5
  CVE-2020-28928      musl         1.1.19-r11    medium      fixed in 1.2.2-r1      5.5
  CVE-2021-42374      busybox      1.28.4-r3     medium      fixed in 1.34.1-r7     5.3

Compliance
==========
  Severity    Description
  --------    -----------
  high        (CIS_Docker_CE_v1.1.0 - 4.1) Image should be created with a non-root user

  Vulnerability
    Critical: 2
    High:     12
    Medium:   4
    Low:      0
    Total:    18

  Compliance
    Critical: 0
    High:     1
    Medium:   0
    Low:      0
    Total:    1

```
